### PR TITLE
Kernel no repeat

### PR DESCRIPTION
**Pull request by loopinf**
## Title

kernel fusion no repeat inference

## Description

-

## Linked Issues

- resolved #00

### CODE REVIEW

File: oslo/torch/_C/csrc/CUDABinder.cpp [hunk 1]
```diff
@@ -0,0 +1,451 @@
+#include <cassert>
```
Comment (hyunwoongko) [inline oslo/torch/_C/csrc/CUDABinder.cpp:1]: What's this?

File: oslo/torch/nn/modules/ngram_repeat_block.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
+# Originally from Microsoft Corporation.
```
Comment (hyunwoongko) [inline oslo/torch/nn/modules/ngram_repeat_block.py:1]: I think we should make the wrapper I said for this.

Comment (loopinf) [reply oslo/torch/nn/modules/ngram_repeat_block.py:1]: Ok I'll try

File: oslo/torch/_C/csrc/CUDABinder.cpp (hunk 1 above)
Comment (loopinf) [reply oslo/torch/_C/csrc/CUDABinder.cpp:1]: I think I should remove it

Comment (loopinf) [reply oslo/torch/_C/csrc/CUDABinder.cpp]: - [x] CUDABinder.cpp removed

File: oslo/torch/nn/modules/ngram_repeat_block.py (hunk 2 above)
Comment (loopinf) [reply oslo/torch/nn/modules/ngram_repeat_block.py:1]: Add wrapper on `kernel_fusion_utils.py`

**Pull request merged**